panther_velodyne/panther_velodyne.launch

```diff
@@ -0,0 +1,66 @@
+<!-- -*- mode: XML -*- -->
+<!-- run velodyne_pointcloud/TransformNodelet in a nodelet manager for a VLP-16 -->
+
+<launch>
+
+  <node name="panther_velodyne_tf" pkg="tf" type="static_transform_publisher" args="0.185 0 0 0 0 0 base_link velodyne 100" />
```

Comment:
```suggestion
  <node name="panther_velodyne_tf" pkg="tf" type="static_transform_publisher" args="0.185 0.0 0.2093 0.0 0.0 0.0 base_link velodyne 100" />
```
base_link is lying on the ground. This will space it the point where LiDAR has its source while being mounted on the rails